tools/cloud-build/daily-tests/tests/hpc-enterprise-slurm.yml

```diff
@@ -22,6 +22,7 @@ zone: europe-west1-d
 cli_deployment_vars:
   region: europe-west1
   zone: "{{ zone }}"
+  enable_cleanup_compute: true
```

Comment:
This is a no-op since this option is already enabled for [this particular blueprint](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/0a301051fb4f9b704861eb2d3f8d5eedf1f7aa67/examples/hpc-enterprise-slurm.yaml#L35C13-L35C13).

I think we wanted to add similar functionality to the test for examples/hpc-slurm.yaml which does not have this setting.

Reply:
Thanks for pointing this out. Updated slurm-v5-hpc-centos7 test variable.